.drone.star

```diff
@@ -1279,7 +1284,12 @@ def ocisService(type, tika_enabled = False, enforce_password_public_link = False
     }
     if type == "app-provider":
         environment["GATEWAY_GRPC_ADDR"] = "0.0.0.0:9142"
-        environment["MICRO_REGISTRY"] = "mdns"
+
+        environment["MICRO_REGISTRY"] = "natsjs"
+        environment["MICRO_REGISTRY_ADDRESS"] = "0.0.0.0:9233"
+        environment["NATS_NATS_HOST"] = "0.0.0.0"
+        environment["NATS_NATS_PORT"] = 9233
```

Comment:
The previous issue was https://github.com/owncloud/web/issues/9687
But now the `mdns` is also pretty much stable. e.g this build https://drone.owncloud.com/owncloud/web/40858/1/1

So, should we leave as it is (use `mdns`) or use `natsjs`?

CC @ScharfViktor @kobergj

Reply:
I really don't know which `MICRO_REGISTRY` should we use

We don't have default value. need to choose (ocis in CI uses `natsjs`)

<img width="812" alt="image" src="https://github.com/owncloud/web/assets/84779829/4a2d5955-6097-4d7d-a279-82c7bb5f599b">

Reply:
I'd be happy if we could use `natsjs`. We want to make this our primary registry so catching problems with it early would be beneficial.

Reply:
Thanks, I will make this PR ready